Neos.Flow/Documentation/conf.py

```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = 'dev-master'
+version = '7.0'
```

Comment:
```suggestion
version = 'dev-7.0'
```
Was that `dev-7.0` intended? Because it should be just 7.0, no?

Reply:
Well, I just copied what @kdambekalns did in 37396baf4725379f7d24146738fe17f1b8a30d4f

Reply:
Rightly so.

Reply:
Ah, that targets the next changed line, actually…